index.html

```diff
@@ -452,18 +452,31 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 used to fetch the resource, as identified by the ALPN Protocol ID
 [[!RFC7301]]; resources retrieved from <a data-cite=
 "!HTML/#relevant-application-cache">relevant application caches</a>
-or local resources, return an empty string. The registry for
-protocol identifiers lists the possible values for the <a href=
+or local resources, return an empty string. When a proxy is
+configured, if a tunnel connection is established then this attribute
+MUST return the ALPN Protocol ID of the tunneled protocol, otherwise
+it MUST return the ALPN Protocol ID of the first hop to the proxy.
+In order to have precisely one way to represent any ALPN protocol ID,
+the following additional constraints apply: octets in the ALPN
+protocol MUST NOT be percent-encoded if they are valid token
+characters except "%", and when using percent-encoding, uppercase hex
+digits MUST be used.</p>
+<p>The registry for protocol identifiers lists the possible values
+for the <a href=
 "https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids">
-ALPN Protocol IDs</a>. When a proxy is configured, if a tunnel
-connection is established then this attribute MUST return the ALPN
-Protocol ID of the tunneled protocol, otherwise it MUST return the
-ALPN Protocol ID of the first hop to the proxy. In order to have
-precisely one way to represent any ALPN protocol ID, the following
-additional constraints apply: octets in the ALPN protocol MUST NOT
-be percent-encoded if they are valid token characters except "%",
-and when using percent-encoding, uppercase hex digits MUST be
-used.</p>
+ALPN Protocol IDs</a>. For the HTTP/0.9 and HTTP/1.0 network
+protocols, which are not defined in the ALPN Protocol ID registry,
+user agents MAY return "http/0.9" and "http/1.0" respectively. In
+case the user agent is using an experimental protocol which ALPN
+protocol ID has not yet been defined in the above registry, the user
+agent SHOULD use the ALPN negotiated value or MAY use another descriptive
+string if one is not available.</p>
```

Comment:
As suggested on the issue:  MUST use the negotiated ALPN token if any, MAY use another descriptive string if ALPN was not used.

Reply:
Changed in that spirit. PTAL